src/Compiler/Driver/CompilerConfig.fsi

```diff
@@ -143,6 +143,18 @@ type VersionFlag =
 
     member GetVersionString: implicitIncludeDir: string -> string
 
+[<RequireQualifiedAccess>]
+type OptimizationDataOptions =
+    | None
```

Comment:
Shouldn't there be a fourth option, which is "an uncompressed resource" (like F# does today by default)?

Reply:
reverted to nosig and noopt, added /compressedmetadata